client/app/components/notification/notification.html

```diff
@@ -0,0 +1,27 @@
+<div layout="row">
+  <div flex></div>
+
+<div id="notification" class="md-whiteframe-2dp" style="margin: 0; padding: 0">
+  <md-toolbar class="md-hue-1">
+    <div class="md-toolbar-tools">
+      <h1 flex>
+        <i class="fa fa-user" aria-hidden="true"></i>&nbsp;
+        {{ $ctrl.userdata.display_name }}
```

Comment:
Maybe add a message here to make clear that the user is on the notifications page.